.github/workflows/ci-cd-trigger.yml

```diff
@@ -136,19 +137,34 @@ jobs:
             preview_explorer=$(printf "https://%s.%s.vega.rocks" "explorer" "$branch_slug")
           fi
           projects="$(echo $projects_e2e | sed 's|-e2e||g')"
-          if echo "$affected" | grep -q multisig-signer; then
-            echo "Tools are affected"
-            # tools are only applicable to check previews or deploy from develop to mainnet
-            if [[ "${{ github.event_name }}" = "pull_request" ]]; then
+          if [[ "${{ github.event_name }}" = "pull_request" ]]; then
+            if echo "$affected" | grep -q multisig-signer; then
+              echo "Tools are affected"
+              # tools are only applicable to check previews or deploy from develop to mainnet
               echo "Deploying tools on preview"
               preview_tools=$(printf "https://%s.%s.vega.rocks" "tools" "$branch_slug")
               projects+=' "multisig-signer" '
             fi
-            if [[ "${{ github.ref }}" =~ .*develop$ ]]; then
+            if echo "$affected" | grep -q static; then
```

Comment:
**thought (non-blocking)**:  I think this logic has got a bit complex in Bash, but already made a ticket to address this separately: https://github.com/vegaprotocol/frontend-monorepo/issues/4072

So I think it's fine as is.